src/modules/decklink/consumer/decklink_consumer.cpp

```diff
@@ -419,7 +440,13 @@ struct decklink_consumer : public IDeckLinkVideoOutputCallback
         }
 #endif
         try {
-            auto tick_time = tick_timer_.elapsed() * format_desc_.fps / field_count_ * 0.5;
+            auto elapsed = tick_timer_.elapsed();
+            int  fieldTimeMs = static_cast<int>(1000 / format_desc_.fps);
+            std::chrono::high_resolution_clock::time_point f2TimePoint =
```

Comment:
Could you add a comment to explain why this is such a complex timepoint to calculate, and a quick justification (to not limit the speed when the trying to refill the decklink frame buffer)